Box.V2.Test/Box.V2.Test.csproj

```diff
@@ -1,40 +1,29 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;netcoreapp2.2;netcoreapp3.1;net5.0;net6.0;net45</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;netcoreapp2.2;netcoreapp3.1;net5.0;net6.0;net462</TargetFrameworks>
```

Comment:
is there a reason why you keep targeting core beyond EOL? Currently, only net6 and net7 are supported, see [here](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core)